application/bots/twitterbot.go

```diff
@@ -66,16 +43,15 @@ type TwitterOAuth struct {
 // tuple. Otherwise, it returns a TwitterBot struct
 // with the appropriate values obtained during the setup.
 func NewTwitterBot(path string) (Bot, error) {
-	var conf TwitterConfig
-	if _, err := toml.DecodeFile(path, &conf); err != nil {
-		return nil, fmt.Errorf("Failed to load config: %v", err)
+	var conf *TwitterConfig = &TwitterConfig{}
+	if err := conf.InitConfig(path); err != nil {
```

Comment:
Please see https://github.com/coniks-sys/coniks-go/pull/195/files#r154509882. Then this could be
```
var config *TwitterConfig = &TwitterConfig{}
config.Load(path)
```